src/main/java/ch/heigvd/amt/stack/infrastructure/persistence/database/JdbcRepository.java

```diff
@@ -27,7 +27,7 @@
             "CREATE TABLE IF NOT EXISTS Session" +
                     "( idSession VARCHAR PRIMARY KEY" +
                     ", idxCredential VARCHAR" +
-                    ", tag VARCHAR(50)" +
+                    ", tag VARCHAR(50) PRIMARY KEY " +
```

Comment:
Why would we need this ?

Reply:
We didn't agreed yesterday that tag is also part of the PK ?

Reply:
No 🙂